packages/default/scss/multiselect/_layout.scss

```diff
@@ -30,7 +30,8 @@
 
                 .k-button {
                     width: auto;
-                    margin: $padding-y-sm $padding-y-sm 0 0;
+                    margin-right: $padding-y-sm;
+                    margin-left: 0;
```

Comment:
How does this look when the selected items (.k-button) wrap on a second line? Add a visual test for that.

Reply:
Those changes are applied only for RTL mode. The idea is to update only horizontal margins for RTL mode. The vertical should be the same.